includes/polling/os/trango.inc.php

```diff
@@ -0,0 +1,3 @@
+<?php
+$version = trim(snmp_get($device, ".1.3.6.1.4.1.5454.1.80.1.1.2.0", "-Oqv", "GIGA-PLUS-MIB"), '"');
+$hardware = "Trango " . trim(snmp_get($device, ".1.3.6.1.2.1.1.1.0", "-Oqv", "GIGA-PLUS-MIB"), '"');
```

Comment:
This variable is used to populate some search fields so I think dropping 'Trango ' should be done.